mongo/integration/retryable_writes_prose_test.go

```diff
@@ -214,4 +217,76 @@ func TestRetryableWritesProse(t *testing.T) {
 				"expected an insert event, got a(n) %v event", cmdEvt.CommandName)
 		}
 	})
+
+	nwpCommandMonitor := new(event.CommandMonitor)
+	nwpClientOpts := options.Client().SetMonitor(nwpCommandMonitor).SetRetryWrites(true)
+
+	mtNWPOpts := mtest.NewOptions().ClientOptions(nwpClientOpts).MinServerVersion("6.0").
+		Topologies(mtest.ReplicaSet)
+
+	mt.RunOpts(fmt.Sprintf("%s label returns original error", driver.NoWritesPerformed), mtNWPOpts,
+		func(mt *mtest.T) {
+			const shutdownInProgressErrorCode int32 = 91
+			const notWritablePrimaryErrorCode int32 = 10107
+
+			// Disable any enabled fail points on exit.
+			defer mt.SetFailPoint(mtest.FailPoint{
```

Comment:
I believe `mt.SetFailPoint` will automatically disable the fail point with `ClearFailPoints` at the end of the test, so it seems like this `defer` is unnecessary.